README.md

```diff
@@ -310,8 +310,8 @@ layout: ?Array<{i?: string, x: number, y: number, w: number, h: number}> = null,
 // Margin between items [x, y] in px.
 margin: ?[number, number] = [10, 10],
 
-// Padding inside the container [x, y] in px
-containerPadding: ?[number, number] = margin,
+// Padding inside the container [x, y] or [top, right, bottom, left] in px
+containerPadding: ?[number, number] | ?[number, number, number, number] = margin,
```

Comment:
This follows the same order as padding in CSS. I kept the order the same when passing two values to avoid breaking changes.